### PR TITLE
CRW-1093 CRW-1124 fix problem with cacert...

### PR DESCRIPTION
**Pull request by nickboldt**
CRW-1093 CRW-1124 fix problem with cacert permission changes

Change-Id: Ifd09eb30732a31bcc0fcf331700a43eed75589b1
Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

**Comment by benoitf** [comment]
just curious, why are you dropping everything from the PR template ?

File: dockerfiles/che/entrypoint.sh [hunk 1]
```diff
@@ -306,7 +306,7 @@ add_cert_to_truststore() {
   echo "$1" > $SELF_SIGNED_CERT
 
   # make sure that owner has permissions to write and other groups have permissions to read
-  chmod 644 $JAVA_TRUST_STORE
+  chmod 644 $JAVA_TRUST_STORE || true
```
Comment (benoitf) [inline dockerfiles/che/entrypoint.sh:310]: should it be true or a message should be logged ?

File: dockerfiles/che/rhel.Dockerfile [hunk 2]
```diff
@@ -34,10 +32,10 @@ RUN mkdir /logs /data && \
     chown -R user /home/user && \
     chmod -R g+rwX /home/user && \
     find /home/user -type d -exec chmod 777 {} \; && \
+    chmod 0777 /home/user/cacerts && \
```
Comment (benoitf) [inline dockerfiles/che/rhel.Dockerfile]: why do we need this kind of permissions on this file ? it looks like a big hole

File: dockerfiles/che/entrypoint.sh (hunk 1 above)
Comment (nickboldt) [reply dockerfiles/che/entrypoint.sh:310]:
```
Using custom assembly from /home/user/codeready
Found a custom cert. Adding it to java trust store /home/user/cacerts
chmod: changing permissions of '/home/user/cacerts': Operation not permitted
```

I suspect even when I use `|| true` we'll get that in the log... we'll just not abort the entrypoint.sh script at that point.

File: dockerfiles/che/rhel.Dockerfile (hunk 2 above)
Comment (nickboldt) [reply dockerfiles/che/rhel.Dockerfile]: I figured doing this would allow the later change to 644 in the entrypoint.sh script

**Comment by nickboldt** [comment]
> just curious, why are you dropping everything from the PR template ?

I generate PRs from a script that calls `hub`, and it apparently ignores the template in the repo.

**Review by benoitf — APPROVED** [review]

**Pull request merged**